pageserver/src/bin/pageserver.rs

```diff
@@ -459,11 +460,18 @@ fn start_pageserver(conf: &'static PageServerConf) -> Result<()> {
         }
     }
 
-    // Initialize tenant manager.
-    tenant_mgr::init(conf);
-
     // keep join handles for spawned threads
-    let mut join_handles = vec![];
+    // don't spawn threads before daemonizing
+    let mut join_handles = Vec::new();
+
+    let storage_accessor = relish_storage::init_storage(conf)?.map(|(accessor, handle)| {
```

Comment:
This is the first, main, upload/download thread that starts with the rest of the pageserver.
I've tried to elaborate on that here: https://github.com/zenithdb/zenith/pull/669#discussion_r716734819

Currently, there's no good way to stop it, similar to the rest of the threads.